<!-- -->
<launch>

  <node pkg="agv_robot" type="pubOdom" name="pubOdom">
  </node>       
      
</launch>
